app/shared/productResponse.ts

```diff
@@ -31,6 +33,21 @@ export const alertTextWithoutCTA = (productDetail: ProductDetail) =>
 export const sortByJoinDate = (a: ProductDetail, b: ProductDetail) =>
   b.joinDate.localeCompare(a.joinDate);
 
+export interface DeliveryRecordsDetail {
```

Comment:
this and the new type above probably should move to `app/client/components/delivery/records/deliveryRecordsApi.ts` since they don't need to be shared with server-side (as far as I can tell) and this `productResponse.ts` just has interfaces relating to `members-data-api` responses IIRC